src/icons/imported/Icon/IconBoxApp512.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+/* eslint-disable react/jsx-sort-props */
+import * as React from 'react';
+import AccessibleSVG from '../../accessible-svg';
+
+type SvgProps = {
+    height?: number,
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconBoxApp512 = (props: SvgProps) => (
+    <AccessibleSVG width={32} height={32} viewBox="0 0 512 512" {...props}>
+        <g fill="none" fillRule="evenodd">
+            <rect fill="#0061D5" width={512} height={512} rx={118} />
```

Comment:
Can we use predefined variables for the color values?

Reply:
Yeah we should use the imported vars. If these are getting auto-generated, then something like this can work

```jsx
import * as colors from '../../styles/variables';

<rect fill={colors.bdlBoxBlue} width={512} height={512} rx={118} />
```

Reply:
I was going back and forth on that for a bit actually. these icons have some colors that are defined, and others that aren't (due to branding restrictions, etc.) I err'd on the side of simplicity since not every one of these icons will include colors that are part of BDL and we don't want to import the color module(s) if it won't get used.

That said, these imports will eventually contain other icons that **should** have swapped values for sure, so I planned to add that in once we get got the first set of non-file/folder icons.

Let me know if you'd rather see that now